.github/workflows/push_ocm.yaml

```diff
@@ -70,3 +62,39 @@ jobs:
             ghcr.io/open-component-model/ocm/ocm.software/ocmcli/ocmcli-image:latest
             ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+  ocm-cli-latest:
+    name: ocm-cli build latest
+    runs-on: large_runner
+    permissions:
+      contents: write
+      id-token: write
+      packages: write
+    steps:
+    - name: Self Hosted Runner Post Job Cleanup Action
+      uses: TooMuch4U/actions-clean@v2.2
+    - name: Generate token
+      id: generate_token
+      uses: tibdex/github-app-token@v2
+      with:
+        app_id: ${{ secrets.OCMBOT_APP_ID }}
+        private_key: ${{ secrets.OCMBOT_PRIV_KEY }}
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        token: ${{ steps.generate_token.outputs.token }}
+    - name: Setup Go
+      uses: actions/setup-go@v5
+      with:
+        go-version-file: '${{ github.workspace }}/go.mod'
+        check-latest: false
+    - name: Run GoReleaser --nightly
+      uses: goreleaser/goreleaser-action@v6
+      with:
+        distribution: goreleaser
+        version: 2.2.0
+        args: release --clean --nightly --timeout 60m --skip=validate --config=.github/config/latest.yml
```

Comment:
Looking at the [references](https://goreleaser.com/customization/nightlies/) it is described as `goreleaser` pro function. Do we have a pro license?

Reply:
damn it... I thought, last time, when I checked, it wasn't a PRO feature...

Reply:
costs 50 Euro per month per Enterprise. We may check if somebody at SAP already has such a license...

Reply:
found another solution... please review again and spend me the 50 bucks ;-)